ush/rocoto/workflow_utils.py

```diff
@@ -33,6 +33,150 @@ def __init__(self,scripts,errors):
+            'noscrub': '/gpfs/hps3/emc/global/noscrub/$USER',
+            'account': 'GFS-DEV',
+            'queue': 'dev',
+            'queue_service': 'dev_transfer',
+            'chgrp_rstprod': 'YES',
+            'chgrp_cmd': 'chgrp rstprod',
+            'hpssarch': 'YES',
+             }
+
+        return info
+
+
+    @property
+    def wcoss_dell_p3(self):
+
+        info = {
+            'base_git': '/gpfs/dell2/emc/modeling/noscrub/emc.glopara/git',
+            'base_svn': '/gpfs/dell2/emc/modeling/noscrub/emc.glopara/git',
+            'dmpdir': '/gpfs/dell3/emc/global/dump',
+            'nwprod': '${NWROOT:-"/gpfs/dell1/nco/ops/nwprod"}',
+            'comroot': '${COMROOT:-"/gpfs/dell1/nco/ops/com"}',
+            'homedir': '/gpfs/dell2/emc/modeling/noscrub/$USER',
+            'stmp': '/gpfs/dell3/stmp/$USER',
+            'ptmp': '/gpfs/dell3/ptmp/$USER',
+            'noscrub': '$HOMEDIR',
+            'account': 'GFS-DEV',
+            'queue': 'dev',
+            'queue_service': 'dev_transfer',
+            'chgrp_rstprod': 'YES',
+            'chgrp_cmd': 'chgrp rstprod',
+            'hpssarch': 'YES',
+             }
+
+        return info
+
+    @property
+    def wcoss_dell_p3p5(self):
+
+        info = {
+            'base_git': '/gpfs/dell2/emc/modeling/noscrub/emc.glopara/git',
+            'base_svn': '/gpfs/dell2/emc/modeling/noscrub/emc.glopara/git',
+            'dmpdir': '/gpfs/dell3/emc/global/dump',
+            'nwprod': '${NWROOT:-"/gpfs/dell1/nco/ops/nwprod"}',
+            'comroot': '${COMROOT:-"/gpfs/dell1/nco/ops/com"}',
+            'homedir': '/gpfs/dell2/emc/modeling/noscrub/$USER',
+            'stmp': '/gpfs/dell3/stmp/$USER',
+            'ptmp': '/gpfs/dell3/ptmp/$USER',
+            'noscrub': '$HOMEDIR',
+            'account': 'GFS-DEV',
+            'queue': 'dev2',
+            'queue_service': 'dev2_transfer',
+            'chgrp_rstprod': 'YES',
+            'chgrp_cmd': 'chgrp rstprod',
+            'hpssarch': 'YES',
+             }
+
+        return info
+
+    @property
+    def hera(self):
+
+        info = {
+            'base_git': '/scratch1/NCEPDEV/global/glopara/git',
+            'base_svn': '/scratch1/NCEPDEV/global/glopara/svn',
+            'dmpdir': '/scratch1/NCEPDEV/global/glopara/dump',
+            'nwprod': '/scratch1/NCEPDEV/global/glopara/nwpara',
+            'comroot': '/scratch1/NCEPDEV/global/glopara/com',
+            'homedir': '/scratch1/NCEPDEV/global/$USER',
+            'stmp': '/scratch1/NCEPDEV/stmp2/$USER',
+            'ptmp': '/scratch1/NCEPDEV/stmp4/$USER',
+            'noscrub': '$HOMEDIR',
+            'account': 'fv3-cpu',
+            'queue': 'batch',
+            'queue_service': 'service',
+            'partition_batch': 'hera',
+            'chgrp_rstprod': 'YES',
+            'chgrp_cmd': 'chgrp rstprod',
+            'hpssarch': 'YES',
+             }
+
+        return info
+
+    @property
+    def orion(self):
```

Comment:
Perhaps these machine blocks could be put in some kind of parm file(s) and these scripts could parse them? An idea for later, not for this PR.